### ISSUE TITLE
Translators missing on the citation page

### ISSUE THREAD

**Issue by wenkokke** [Agda-zh/PLFA-zh#65]
I noticed that on [the citation page](https://agda-zh.github.io/PLFA-zh/Citing/), the references and the BibTeX do not include the translators. I feel you should add these! This might be as simple as adding translators in the appropriate places, as I've done below, but it probably makes sense to translate the titles in their entirety.

> Wadler, Philip and Wen Kokke. Programming Language Foundations in Agda. Translated by Fangyi Zhou, Oling Cat, and Rongxiao Fu. Available at https://agda-zh.github.io/PLFA-zh/. 2019.

```bibtex
@Book{plfa2019,
  author     = {Philip Wadler and Wen Kokke},
  title      = {Programming Language Foundations in {A}gda},
  translator = {Fangyi Zhou and Oling Cat and and Rongxiao Fu},
  note       = {Available at \url{https://agda-zh.github.io/PLFA-zh/}},
  year       = 2019,
}
```

**Comment by OlingCat** [comment]
Ah... thank you! I just noticed that.

This issue will be solved after #64, cause we are sync PLFA-zh with `plfa.github.io` for Agda 2.6.0, it has a lot of work. 😅

In Chinese, the cite of PLFA-zh should be

> 《编程语言基础：Agda 语言描述》，Philip Wadler 与 Wen Kokke 著，Fangyi Zhou、Oling Cat 与 Rongxiao Fu 译。可访问 https://agda-zh.github.io/PLFA-zh/ 阅读。

By the way, your name "Wen Kokke" can be transliterated into "温·考克" in Chinese, and "Philip Wadler" should be "飞利浦·瓦德勒".

@roger-uw @fangyi-zhou 话说咱还不知道乃们的中文名字= =？

**Comment by fangyi-zhou** [comment]
周方易

**Comment by roger-uw** [comment]
傅荣枭
________________________________

**Comment by wenkokke** [comment]
> This issue will be solved after #64, cause we are sync PLFA-zh with `plfa.github.io` for Agda 2.6.0, it has a lot of work. 😅

Ah, yeah, I'm sorry for that 😅
